src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesRequest.cs

```diff
@@ -5,7 +5,8 @@ namespace Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancie
 {
     public class SearchApprenticeshipVacanciesRequest : IRequest<SearchApprenticeshipVacanciesResponse>
     {
-        public IEnumerable<string> StandardCodes { get; set; }
+        public List<string> StandardCodes { get; set; } = new List<string>();
```

Comment:
We shouldn't really need to do this default assignment. I know it stops null reference.

Reply:
I tend to disagree